Formula/flow-cli.rb

```diff
@@ -0,0 +1,19 @@
+class FlowCli < Formula
+  desc "Command-line interface that provides utilities for building Flow applications"
+  homepage "https://onflow.org"
+  url "https://github.com/onflow/flow-cli/archive/v0.12.1.tar.gz"
+  sha256 "ad231ca19296ca65522eb71115900c9bd6918335caafef5913689acc1ec7effe"
+  license "Apache-2.0"
+  head "https://github.com/onflow/flow-cli.git"
+
+  depends_on "go@1.14" => :build
```

Comment:
This should be the latest go.

Reply:
I've changed this to the latest version of Go in c4cf1e6